pkgs/top-level/python-packages.nix

```diff
@@ -3074,35 +3071,35 @@ in {
   };
 
   botocore = buildPythonPackage rec {
-    version = "1.4.87"; # This version is required by awscli
-    name = "botocore-${version}";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/b/botocore/${name}.tar.gz";
-      sha256 = "0fga1zjffsn2h50hbw7s4lcv6zwz5dcjgvjncl5y392mhivlrika";
-    };
+      version = "1.4.92"; # This version is required by awscli
```

Comment:
We usually indent nix by two spaces, not four spaces. Please restore the original indentation.